ddterm/shell/extension.js

```diff
@@ -140,6 +142,39 @@ class ExtensionDBusInterface {
     }
 }
 
+class DesktopEntry {
+    constructor() {
+        this.file = Gio.File.new_for_path(GLib.build_filenamev(
+            [
+                GLib.get_user_data_dir(),
+                'applications',
+                `${APP_ID}.desktop`,
+            ]
+        ));
+    }
+
+    install() {
+        try {
+            Gio.File.new_for_path(GLib.build_filenamev(
```

Comment:
Nitpick: maybe you could create this Gio.File in constructor too? Less indentation -> easier to read

Reply:
Done :)